Sources/AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -30,6 +30,8 @@
 --typeblanklines preserve # blankLinesAtStartOfScope, blankLinesAtEndOfScope
 --emptybraces spaced # emptyBraces
 --someAny disabled # opaqueGenericParameters
+--elseposition same-line #elseOnSameLine
+--elseblankline preserve #elseOnSameLine
```

Comment:
This new option implemented in https://github.com/nicklockwood/SwiftFormat/pull/1447